src/components/SpecificArticle.jsx

```diff
@@ -31,6 +31,7 @@ const SpecificArticle = () => {
           <div data-cy="article">
             <h1 data-cy="title">{article.title}</h1>
             <h3 data-cy="teaser">{article.teaser}</h3>
+            <Image size="medium" centered data-cy="image" src={article.image} />
```

Comment:
please make a conditional here also, like you have for the article list

Reply:
Done!